api/src/main/java/com/github/retrooper/packetevents/protocol/chat/ChatType.java

```diff
@@ -27,12 +27,14 @@ public enum ChatType {
     TEAM_MSG_COMMAND,
     EMOTE_COMMAND,
     TELLRAW_COMMAND;
-    
+
+    public static final ChatType[] VALUES = values();
```

Comment:
This may not be public as that would expose the values array and allow for mutations to it